lib/eslint/eslint.js

```diff
@@ -168,6 +169,7 @@ function processOptions({
     resolvePluginsRelativeTo = null, // ← should be null by default because if it's a string then it suppresses RFC47 feature.
     rulePaths = [],
     useEslintrc = true,
+    warnIgnored = true,
```

Comment:
This will override `default-cli-options.js`, just like all of the options above this line.